en_US/observability/alarms.md

```diff
@@ -1,61 +1,57 @@
 # Alarm
 
-EMQX Broker has built-in monitoring and alarm functionality. Currently it supports monitoring of CPU occupancy, system and process memory occupancy, number of processes, rule engine resource status, cluster partition and healing, and it can alarm on these metrics. Both activation and deactivation of alarms will generate an alarm log and the Broker will publish an MQTT message with the topic of `$SYS/brokers/<Node>/alarms/activate` or `$SYS/brokers/<Node>/alarms/deactivate`. Users can subscribe to the topics of `$SYS/brokers/+/alarms/avtivate` and `$SYS/brokers/+/alarms/deactivate` to get alarm notifications.
+EMQX has offered a built-in monitoring and alarm functionality for monitoring the CPU occupancy, system and process memory occupancy, number of processes, rule engine resource status, cluster partition and healing, and will raise an alarm in case of system malfunctions. The activation and deactivation of alarms will generate an alarm log and EMQX will publish an MQTT message with the topic of `$SYS/brokers/<Node>/alarms/activate` or `$SYS/brokers/<Node>/alarms/deactivate`. Users can subscribe to the topics of `$SYS/brokers/+/alarms/activate` and `$SYS/brokers/+/alarms/deactivate` to receive alarm notifications.
```

Comment:
```suggestion
EMQX has offered a built-in monitoring and alarm functionality for monitoring the CPU occupancy, system and process memory occupancy, number of processes, rule engine resource status, cluster partition and healing, and will raise an alarm in case of system malfunctions. The activation and deactivation of alarms will generate an alarm log and EMQX will publish an MQTT message with the topic of `$SYS/brokers/<Node>/alarms/activate` or `$SYS/brokers/<Node>/alarms/deactivate`. Users can subscribe to the topic to receive alarm notifications.
```